src/rpc/multipass.proto

```diff
@@ -139,8 +141,17 @@ message FindReply {
         string version = 3;
         repeated AliasInfo aliases_info = 4;
     }
-    repeated ImageInfo images_info = 1;
-    string log_line = 2;
+    message BlueprintInfo {
+        string os = 1;
+        string release = 2;
+        string version = 3;
+        repeated AliasInfo aliases_info = 4;
+    }
+    bool show_images = 1;
```

Comment:
Added flags here which are persisted in `daemon.cpp` so that output can appropriately reflect the flags used in the request. Is there a better way?